### PR TITLE
Setup project

### PR DESCRIPTION
**Pull request by thachnguyendino**
Changes:
- Set package name: com.dinosys.sportbook
- Apply Support from API 15 and higher.
- Added library: ButterKnife, Retrofit, RxAndroid, Glide, Gson, Facebook sdk.

### CODE REVIEW

**Comment by ducdino** [comment]
I have reviewed this. @ndduy Please approve this pull request.

File: SportBook/.idea/compiler.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      <entry name="!?*.clj" />
+      <entry name="!?*.aj" />
+    </wildcardResourcePatterns>
+    <annotationProcessing>
+      <profile default="true" name="Default" enabled="false">
+        <processorPath useClasspath="true" />
+      </profile>
+    </annotationProcessing>
+  </component>
+</project>
```
Comment (ndduy) [inline SportBook/.idea/compiler.xml]: Please add .idea into git ignore

File: SportBook/app/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+/build
```
Comment (ndduy) [inline SportBook/app/.gitignore]: We should have 1 gitignore only

File: SportBook/.gitignore [hunk 3]
```diff
@@ -0,0 +1,9 @@
+*.iml
+.gradle
+/local.properties
+/.idea/workspace.xml
+/.idea/libraries
+.DS_Store
+/build
+/captures
+.externalNativeBuild
```
Comment (ndduy) [inline SportBook/.gitignore:9]: Example of gitignore

# Built application files
*.apk
*.ap_

# Files for the Dalvik VM
*.dex

# Java class files
*.class

# Generated files
bin/
gen/

# Gradle files
.gradle/
build/

# Local configuration file (sdk path, etc)
local.properties

# Proguard folder generated by Eclipse
proguard/

# Log Files
*.log

# Android Studio Navigation editor temp files
.navigation/

# Android Studio captures folder
captures/

# idea files
.idea/

*.iml

third_parties/android_dbinspector/

**Review by ndduy — CHANGES_REQUESTED** [review]

**Review by ndduy — APPROVED** [review]

**Pull request merged**